tensorflow/core/util/sparse/sparse_tensor.h

```diff
@@ -530,6 +537,65 @@ std::vector<SparseTensor> SparseTensor::Split(const SparseTensor& input_tensor,
   return output_tensors;
 }
 
+template <typename T>
+SparseTensor SparseTensor::Slice(const SparseTensor& input_tensor,
```

Comment:
can you add a C++ test?  i believe there's a util/sparse/sparse_tensor_test.cc where you can add one.